3.interpret_models/log_reg_plate_3_3p_cp_renorm_fs_data/feature_importances/nbconverted/interpret.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Determine the best features using a Logistic Regression Model
+
+# ## Imports
+
+# In[ ]:
+
+
+import sys
+import pandas as pd
+import numpy as np
+from pathlib import Path
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+
+# ## Find the git root Directory
+
+# In[ ]:
+
+
+# Get the current working directory
+cwd = Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
```

Comment:
given the amount of time this line is code is repeated. Considered creating a function `project_dir_check()` in the future

Reply:
I could do something like that, but I would have to create a file with the function inside, and then try to find that file. Although, I could just duplicate that file, so it is always in the same directory as the code. I'll keep that in mind.